docs/plugin-center/observability/metrics-plugin.md

```diff
@@ -169,7 +169,9 @@ Users need to install `Prometheus` service to collect
      - targets: ['localhost:8090'] # metrics of apache shenyu are exposed on port 8090 by default
  ```
 
-* After the configuration is completed, you can directly double-click `prometheus.exe` in the window to start. The default boot port is `9090`, Success can be verified at http://localhost:9090/
```

Comment:
BTW, can you help to add the guide of linux/macos ?

Reply:
yes，I can . I deployed it on macos when I tested it. 
I will submit this part to this pr as soon as possible.